rules/linux/command_and_control_tunneling_via_earthworm.toml

```diff
@@ -20,7 +20,12 @@ license = "Elastic License v2"
 name = "Potential Protocol Tunneling via EarthWorm"
 note = """## Setup
 
-If enabling an EQL rule on a non-elastic-agent index (such as beats) for versions <8.2, events will not define `event.ingested` and default fallback for EQL rules was not added until 8.2, so you will need to add a custom pipeline to populate `event.ingested` to @timestamp for this rule to work.
+If enabling an EQL rule on a non-elastic-agent index (such as beats) for versions <8.2,
```

Comment:
This rule relies on data coming from either Elastic Defend or Auditbeat integrations.